### PR TITLE
Event Site Update (dbt Learns)

### PR DESCRIPTION
**Pull request by fatimaconteh**
Adding q2 learns to the website

### CODE REVIEW

File: _data/events.yml [hunk 1]
```diff
@@ -5,6 +5,59 @@
...
+  start: June 3, 2021
+  location: Distributed
+  url: https://www.tickettailor.com/events/fishtownanalytics/486756
+  description: "dbt Learn is a two-day training program that teaches analysts the tools and knowledge needed to own the entire analytics engineering workflow."
+
+- name: "dbt Learn (UTC-7/West Coast)"
+  type: Education
+  start: May 19, 2021
+  location: Distributed
+  url: https://www.tickettailor.com/events/fishtownanalytics/486755
```
Comment (clrcrl) [inline _data/events.yml:28]:
```suggestion
  url: https://www.tickettailor.com/events/fishtownanalytics/486755
  image: https://i.imgur.com/p2MYMT5.jpg
```

File: _data/events.yml [hunk 2]
```diff
@@ -5,6 +5,59 @@
...
+  start: June 16, 2021
+  location: Distributed
+  url: https://www.tickettailor.com/events/fishtownanalytics/486757
+  description: "dbt Learn is a two-day training program that teaches analysts the tools and knowledge needed to own the entire analytics engineering workflow."
+
+- name: "dbt Learn (UTC+1/London)"
+  type: Education
+  start: June 3, 2021
+  location: Distributed
+  url: https://www.tickettailor.com/events/fishtownanalytics/486756
```
Comment (clrcrl) [inline _data/events.yml:20]:
```suggestion
  url: https://www.tickettailor.com/events/fishtownanalytics/486756
  image: https://i.imgur.com/p2MYMT5.jpg
```

File: _data/events.yml [hunk 3]
```diff
@@ -5,6 +5,59 @@
 #   url: external url
 #   image: external url
 #   description: sentence or two (max 60 words)
+- name: "dbt Learn (UTC-4/East Coast)"
+  type: Education
+  start: June 16, 2021
+  location: Distributed
+  url: https://www.tickettailor.com/events/fishtownanalytics/486757
```
Comment (clrcrl) [inline _data/events.yml:12]:
```suggestion
  url: https://www.tickettailor.com/events/fishtownanalytics/486757
  image: https://i.imgur.com/p2MYMT5.jpg
```

**Review by clrcrl — CHANGES_REQUESTED** [review]

**Review by clrcrl — APPROVED** [review]
I had to add some missing images for this to work. No problem at all , just wanted to give you a heads up for next time!

**Comment by fatimaconteh** [comment]
ooo so sorry about that! thanks for adding them in. :)

**Pull request merged**